src/main/java/hackaton/cuckoodobot/Main.java

```diff
@@ -14,6 +21,10 @@
 public class Main {
 
     private final static String BOT_TOKEN = "BOT_TOKEN";
+    private final static String DB_URL = "jdbc:h2:~/cuckoodo;CIPHER=AES"; //db file will be created in home dir with AES encryption
```

Comment:
I think GitHub flow based on 'only-forward' approach. 
With new commits the review works fine.